features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mysql.sql

```diff
@@ -2190,6 +2190,19 @@ CREATE TABLE IF NOT EXISTS AM_WEBHOOKS_UNSUBSCRIPTION (
             ADDED_AT TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
 )ENGINE INNODB;
 
+CREATE TABLE IF NOT EXISTS AM_API_ENVIRONMENT_KEYS
+(
+    ID              INTEGER      NOT NULL AUTO_INCREMENT,
+    UUID            VARCHAR(45)  NOT NULL,
+    ENVIRONMENT_ID  VARCHAR(45)  NOT NULL,
+    API_ID          INTEGER NOT NULL,
```

Comment:
Shall we use the API_UUID itself here which is already unique so we can use it for the foreign key too.. it will simplify quering as well.